api/src/main/java/com/okta/idx/sdk/api/client/IDXAuthenticationWrapper.java

```diff
@@ -395,6 +396,33 @@ public AuthenticationResponse enrollAuthenticator(ProceedContext proceedContext,
         }
     }
 
+    /**
+     * Verify Authenticator with the supplied authenticator options.
+     *
+     * @param proceedContext the ProceedContext
+     * @return the Authentication response
+     */
+    public AuthenticationResponse verifyAuthenticator(ProceedContext proceedContext) {
```

Comment:
This name seems a little confusion to me. It looks like this is doing `poll`, but it's called verifyAuthenticator. It's not clear to me by documentation when I'd use this method versus the other `verifyAuthenticator` method.